beacon-chain/sync/rpc_goodbye.go

```diff
@@ -65,7 +65,12 @@ func (s *Service) disconnectBadPeer(ctx context.Context, id peer.ID, badPeerErr
 		log.WithError(err).Debug("Error when disconnecting with bad peer")
 	}
 
-	log.WithError(badPeerErr).WithField("peerID", id).Debug("Initiate peer disconnection")
+	log.WithError(badPeerErr).
+		WithFields(logrus.Fields{
+			"peerID": id,
+			"agent":  agentString(id, s.cfg.p2p.Host()),
+		}).
+		Debug("Initiate peer disconnection")
```

Comment:
This message isn't clear to me. How about this?


```suggestion
		Debug("Sent peer disconnection")
```

I feel like this message is in the past tense so "sent" might make more sense.